src/constants/styling/buttons.scss

```diff
@@ -9,6 +9,8 @@
   color: $Cararra;
   cursor: pointer;
   border-radius: 5px;
+  margin-right: 9px;
+  margin-left: 0.5rem;
```

Comment:
These changes you have made will affect all primary buttons. Have you checked that the alignment of other buttons is still okay?